src/Magnum/GL/Buffer.h

```diff
@@ -1171,10 +1194,15 @@ class MAGNUM_GL_EXPORT Buffer: public AbstractObject {
          * @overload
          * @m_since{2019,10}
          */
-        template<class T> Buffer& setSubData(GLintptr offset, std::initializer_list<T> data) {
+        template<class T> Buffer& setSubData(GLintptr offset, Containers::ArrayView<T> data) {
             return setSubData(offset, {data.begin(), data.size()});
         }
 
+        /** @overload */
+        template<class T> Buffer& setSubData(GLintptr offset, std::initializer_list<T> data) {
```

Comment:
All these changes to `setData()` and `setSubData()` are unnecessary -- there are `Containers::ArrayView<const void>` overloads right above that takes care of everything. Revert please ;)

Reply:
Done